home/tests.py

```diff
@@ -423,3 +423,10 @@ def test_process_request_with_logged_out_user(self):
         request_mock = Mock(user=self.LoggedOutUserMock())
         pm.process_view(request_mock, None, None, None)
         assert request_mock.profile is None
+
+
+class TestAboutPage:
+    def test_about_page_url(self, client):
+        url = reverse('about')
+        response = client.get(url)
+        assert response.status_code == 200
```

Comment:
LGTM ,
just maybe move this class to another file (according to the new arrangement)